addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3785,8 +3785,12 @@ const SnippetOptionWidget = Widget.extend({
                 // Disable all transitions for the duration of the style check
                 // as we want to know the final value of a property to properly
                 // update the UI.
+                this.options.wysiwyg.odooEditor.observerUnactive('o_we_force_no_transition');
                 this.$target[0].classList.add('o_we_force_no_transition');
-                const _restoreTransitions = () => this.$target[0].classList.remove('o_we_force_no_transition');
+                const _restoreTransitions = () => {
+                    this.$target[0].classList.remove('o_we_force_no_transition');
+                    this.options.wysiwyg.odooEditor.observerActive('o_we_force_no_transition');
+                }
```

Comment:
In the future I guess it would make sense to just shut down the observer for the duration of all `_computeWidgetState` calls. Anyways:
- Should not this be in its own commit with its own explanation + should not it target 15.0?
- Missing `;`
- To be extra careful in stable, I think I would rather unobserve/reobserve two times: around the class addition and around the class removal.